File: DivyaPOC/app/src/main/java/android/divya/com/divyapoc/MainActivity.java
package android.divya.com.divyapoc;

import android.divya.com.divyapoc.adapter.TabViewPagerAdapter;
import android.os.Bundle;
import android.support.design.widget.TabLayout;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity{
    private ViewPager mViewPager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TabLayout tabLayout = (TabLayout) findViewById(R.id.layout_tabs);
        tabLayout.addTab(tabLayout.newTab().setText(R.string.sports));
        tabLayout.addTab(tabLayout.newTab().setText(R.string.tech));
        tabLayout.addTab(tabLayout.newTab().setText(R.string.design));
        tabLayout.addTab(tabLayout.newTab().setText(R.string.politics));
        tabLayout.addTab(tabLayout.newTab().setText(R.string.movie));
        tabLayout.setTabGravity(TabLayout.GRAVITY_FILL);

        setUpTabsAndViewPager(tabLayout);
    }

    private void setUpTabsAndViewPager(final TabLayout tabLayout) {


        mViewPager = (ViewPager) findViewById(R.id.viewpager);
        TabViewPagerAdapter tabViewPagerAdapter = new TabViewPagerAdapter(getSupportFragmentManager(), tabLayout.getTabCount());
        mViewPager.setAdapter(tabViewPagerAdapter);
        mViewPager.addOnPageChangeListener(new TabLayout.TabLayoutOnPageChangeListener(tabLayout));

        tabLayout.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
            @Override
            public void onTabSelected(TabLayout.Tab tab) {
                mViewPager.setCurrentItem(tab.getPosition());
            }

            @Override
            public void onTabUnselected(TabLayout.Tab tab) {

            }

            @Override
            public void onTabReselected(TabLayout.Tab tab) {

            }
        });
    }

}

File: Login/app/src/main/java/github/android/com/loginkotlin/MainActivity.kt
package github.android.com.loginkotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.ProgressBar
import android.widget.Toast

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val etName: EditText = findViewById(R.id.et_name)
        val etPswd: EditText = findViewById(R.id.et_pswd)
        val btn : Button = findViewById(R.id.button)
        val prgs:ProgressBar = findViewById(R.id.progressBar);

        btn.setOnClickListener(View.OnClickListener {
           prgs.visibility
            val text:String = etName.text.toString()
            val pswd:String = etPswd.text.toString()
            Toast.makeText(applicationContext,"Text ->${text}, Password ->${pswd}",Toast.LENGTH_LONG).show()
        })
    }
}
